### DISCUSSION TITLE
Fprime SDR Integration

### DISCUSSION THREAD

**Discussion by satzguyz** [nasa/fprime#1375] [Q&A]
Hello,

Is Fprime already per-integrated with any of the commercially available SDR or any such samples. Instead of using local host, how to use it for Over the air testing.

**Comment by kitty99099** [comment]
Hi! Does anyone have idea about it?

**Comment by timcanham** [comment]
We aren't aware of any SDRs, but that doesn't mean they don't exist. We don't maintain any components for them. If anyone in the community has done it, feel free to contribute them to [F Prime Community](https://github.com/orgs/fprime-community/repositories).

**Reply by kitty99099** [reply]
Thank you for your reply @timcanham . I'll try that. If I have further progress, I'll let youk know and share to the community.
